### DISCUSSION TITLE
Sign-up and sign-in with Google.

### DISCUSSION THREAD

**Discussion by ghost** [monkeytypegame/monkeytype#891] [Ideas] [answered]
Hey Monkeytype contributors, I would love it if you have an option to sign-up with Google. There are many good reasons why this is useful.

- You don't have to verify your email.
- You don't have to think up or generate a new password for this website.
- You don't need to keep the login details safe or on the browser. 
- Better security because there's no new password that can get lost or stolen.
- You can sign-up faster because account creation is just a few clicks.
- You can sign-in faster, because it is also a few clicks.
- You can know you have registered with Monkeytype years later, by checking Myaccount Google.
- Google has two-factor authentication, and better security.

**Comment by Miodec** [comment, marked as answer]
Added

**Reply by ghost** [reply]
That's fantastic! I have issues using it to sign in, I know it is still a work in progress, but could you also add a sign-up with google button? I feel like that might be more intuitive. Anyways good on you for working on this!

**Reply by Miodec** [reply]
Whats not working? It was fine for me when i tested it

**Reply by ghost** [reply]
A firebase redirection happens for a short while then it goes back to the app and an Error pops up saying the identity provider configuration is disabled.

**Reply by Miodec** [reply]
My bad, should be enabled now.

**Reply by ghost** [reply]
Ok it works now!
